README.md

```diff
@@ -56,36 +58,40 @@ const order = await rarible.createOrder({
 console.log(order); // {..., hash: "abc123"}
 
 
-// Update that sell order to do a "flash sale":
-//  - Lower price from 1.75 ETH to 1.5 ETH
-await rarible.updateOrder({
-    hash: "abc123",
+//  Update sell order, and lower price from 1.75 ETH to 1.5 ETH
+order = await rarible.updateOrder({
+    order: order
     amount: "1500000000000000000",
-    endBlockTimestamp: 120324374
```

Comment:
I've removed this like because of the previous feedback, but i still think that we should, in theory, allow to update orders like this.

update order, and have all/almost all parameters as optional input.